fix8.sh

```
@@ -0,0 +1 @@
+sed -i '175c\    if not re.match(r"^[a-zA-Z0-9.-]+$", str(host)):\n        logger.error(f"Invalid host parameter: {host}")\n        return' setup/services.py
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)